src/components/VideoDetails/VideoDetails.component.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import { useYouTube } from '../../providers/YouTubeAPI';
+import { Title, Details, VideoDetailsContainer } from './VideoDetails.styled';
+
+function VideoDetails({ videoId }) {
+  const [details, setDetails] = useState({ title: '', description: '' });
+  const { getVideoDetails } = useYouTube();
+
+  useEffect(() => {
+    getVideoDetails(videoId).then(
+      ({
+        items: [
+          {
+            snippet: { title, description },
+          },
+        ],
+      }) => {
+        setDetails({ title, description });
+      }
+    );
+  }, [getVideoDetails, videoId]);
+
+  const { title, description } = details;
+
+  return (
+    <VideoDetailsContainer>
+      <Title aria-label="title">{title}</Title>
+      <Details
+        role="region"
+        aria-label="description"
+        dangerouslySetInnerHTML={{ __html: description.replace(/\n/g, '</br>') }}
```

Comment:
Details could be a component by itself that parses the response, so you don't have to use dangerouslySetInnerHTML

Reply:
on it, thanks 😄 👍